cli/ceph/orch/orch.py

```diff
@@ -0,0 +1,36 @@
+from cli import Cli
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class Orch(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(Orch, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} orch"
+
+    def ls(self, **kw):
```

Comment:
Don't use `kw`, pass variables directly and operate optional parameters accordingly

Reply:
Just to clarify, the method takes service_type as iscsi, and format as json-pretty 

If we look at ceph orch ls, it accepts few are other params also. 

The command looks like
"ceph orch ls --service_type iscsi --format json-pretty"

So you are suggesting to modify the current method to 

def ls (self, service_type=None, service_name=None, export=None, format=None, refresh=None) ?

Also if a new support is added to this ls later, we might have to change the function definition right? So wouldn't it be better to pass it as kwargs itself?